internal/conf/conf.go

```diff
@@ -36,17 +51,38 @@ type PerfConfig struct {
 	NodeURLs         []string
 	StackJSONPath    string
 	DelinquentAction string
+	Daemon           bool
+	SenderURL        string
+}
+
+type PerformanceTestConfig struct {
+	StackJSONPath string           `json:"stackJSONPath" yaml:"stackJSONPath"`
+	Instances     []InstanceConfig `json:"instances" yaml:"instances"`
+	WSConfig      FireFlyWsConf    `json:"wsConfig,omitempty" yaml:"wsConfig,omitempty"`
+	Daemon        bool             `json:"daemon,omitempty" yaml:"daemon,omitempty"`
+}
+
+type InstanceConfig struct {
+	Name            string           `yaml:"name" json:"name"`
+	Tests           []fftypes.FFEnum `yaml:"tests" json:"test"`
+	Length          time.Duration    `yaml:"length" json:"length"`
+	MessageOptions  MessageOptions   `json:"messageOptions,omitempty" yaml:"messageOptions,omitempty"`
+	Sender          int              `json:"sender" yaml:"sender"`
+	Recipient       *int             `json:"recipient,omitempty" yaml:"recipient,omitempty"`
+	TokenOptions    TokenOptions     `json:"tokenOptions,omitempty" yaml:"tokenOptions,omitempty"`
+	ContractOptions ContractOptions  `json:"contractOptions,omitempty" yaml:"contractOptions,omitempty"`
+	Workers         int              `json:"workers" yaml:"workers"`
 }
 
 type FireFlyWsConf struct {
```

Comment:
Could replace this w/ `WSConn` from https://github.com/hyperledger/firefly/pull/738